data/figures.yml

```diff
@@ -120,7 +120,7 @@ figure_list:
     caption: "**`layout: entry`** <br /> **`presentation: landscape`** (default) <br /> An entry page with entry text below and a nearly full-frame image viewer that supports zooming images, maps and videos."
   - id: "modern-entry-side"
     src: /screenshots/modern-entry-side.png
-    caption: "**`layout: entry`** <br /> **`presentation: side-by-side`** <br /> An entry page with scrolling entry text to the right side and a fixed-position image viewer to the left. All entry pages can be made to be in the side-by-side configuration by including `entryPageSideBySideLayout: true` in the config.yaml file of the project."
+    caption: "**`layout: entry`** <br /> **`presentation: side-by-side`** <br /> An entry page with scrolling entry text to the right side and a fixed-position image viewer to the left. All entry pages can be made to be in the side-by-side configuration by including `sidebySide: true` in the `config.yaml` file of the project."
```

Comment:
The B in `sidebySide: true` should be capitalized: `sideBySide: true`.

That said, when referring to configuration values that are nested like this one is, it might be more correct to write it as `entryPage.sideBySide: true`, with the parent value separated by a period. Similar to how we might use the full file path when referring to a file `content/_data/config.yaml`. However, this might be confusing for folks, so I only mention it as an option.

Reply:
@geealbers. I wondered what the best way to handle this was. I think I am going to leave it alone for now, especially because I always have visual examples, and you're right that the longer version might be confusing. That said, I'm going to create a new issue to go back and look at the formatting across the docs once everything major is complete.